<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Video converter (ffmpeg GUI)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButtonSelectFile">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonConvert">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>94</y>
      <width>321</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>63</y>
      <width>321</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>40</y>
      <width>120</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Select output format:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
